### ISSUE TITLE
Warning lines for instances not in inventory

### ISSUE THREAD

**Issue by rrossouw01** [mludvig/aws-ssm-tools#29]
This warning can be quite annoying if you have many instances not in inventory.  It shows not just on a list but also ssh-ssm utils etc

```
$ ssm-session --list --profile admin-guideitprod
[ssm-session] WARNING: SSM inventory entity not recognised: {'Id': 'i-02a#############c4b', 'Data': {}}
....
```

I changed the warning to debug temporarily maybe you want to make that permanent. Or if better option let me know.

```
$ diff /usr/local/lib/python3.8/dist-packages/ssm_tools/resolver.py /tmp/resolver.py
39a40
>
61,62c62
<                     #logger.warning("SSM inventory entity not recognised: %s", entity)
<                     logger.debug("SSM inventory entity not recognised: %s", entity)
---
>                     logger.warning("SSM inventory entity not recognised: %s", entity)
```

**Comment by mludvig** [comment]
Fixed in 4c808c1, thanks for the report.

I'm intrigued when does it happen though? I've got instances that are in SSM as well as many that are not and have never seen this error myself.

**Comment by arthurio** [comment]
@mludvig I have no idea what makes some instances show up in that list of "SSM inventory entity not recognised" either but I do see quite a few that are from a separate EKS cluster (no ssm installed on them), but I don't see "all" of them which doesn't really make sense ¯\_(ツ)_/¯ And they are also terminated instances. Anyway, thanks for the log level change here 👍🏻
